js/client/modules/@arangodb/testing.js

```diff
@@ -195,9 +197,37 @@ const YELLOW = require('internal').COLORS.COLOR_YELLOW;
 
 let failedRuns = {
 };
+
 let allTests = [
 ];
 
+// /////////////////////////////////////////////////////////////////////////////
```

Comment:
@fceller can't we move this next to `js/client/modules/@arangodb/test-utils.js:filterTestcaseByOptions()` since all other filtering is implemented there?

Reply:
Let me check